docs/monthly-meeting/2023-12.md

```diff
@@ -0,0 +1,79 @@
+# Documentation Community Team Meeting (December 2023)
+
+- **Date:** 2023-12-05
+- **Time:** [20:00 UTC](https://arewemeetingyet.com/UTC/2023-12-05/20:00/Docs%20Meeting)
+- **This HackMD:** https://hackmd.io/@encukou/pydocswg1
+- [**Discourse thread**](https://discuss.python.org/t/documentation-community-meeting-tuesday-december-5-2023/40119) (for December)
+- [**Meeting reports**](https://docs-community.readthedocs.io/en/latest/monthly-meeting/) (the latest one might be an [**unmerged PR**](https://github.com/python/docs-community/pulls))
+- **Calendar event:** (send your e-mail to Mariatta for an invitation)
+- **How to participate:**
+  -  Go to [Google Meet](https://meet.google.com/dii-qrzf-wkw) and ask to be let in.
+  -  To edit notes, click the “pencil” or “split view” button on the [HackMD document](https://hackmd.io/@encukou/pydocswg1). You need to log in (e.g. with a GitHub account).
+
+By participating in this meeting, you are agreeing to abide by and uphold the [PSF Code of Conduct](https://www.python.org/psf/codeofconduct/).
+Please take a second to read through it!
+
+## Roll call
+
+(Name / `@GitHubUsername` *[/ Discord, if different]*)
+- Petr Viktorin / `@encukou`
+- Daniele Procida / `@danieleprocida`
+- Hugo van Kemenade / `@hugovk`
+- Jim DeLaHunt / `@JDLH`
+- Ryan Duve / `@ryan-duve`
+- Ned Batchelder / `@nedbat`
+- Bradley / `@shenanigansd`
+- Ezio Melotti / `@ezio-melotti`
+- CAM Gerlach / `@CAM-Gerlach`
+- Usman Akinyemi / `@Unique-Usman`
+
+## Discussion
+
+* [Ryan] standard library documentation reference and adding types to it in a structured way.  For example, `collections.Counter.most_common` could be written
+
+      most_common(n: int) -> list[tuple[Any, int]]
+
+  [Petr] ... or `list[tuple[KeyType, int]]`?
+
+  - this has been discussed. There was opposition to adding the types to the stdlib code itself.
```

Comment:
Is there a source or a link to the discussion? It'd be nice if we could add it here.

Reply:
@hugovk

Reply:
I don't know where it is.

Reply:
@encukou

Reply:
Can't find it on Discourse.  I guess it was too long ago, and might need new discussion -- the situation has changed in the mean time :)
Let me know if you want to work on a proposal, I can dig through mailing list archives from the 2010's for links to past discussions.

Reply:
Yeah, I will like to @encukou. Thanks.